environment/directories.rst

```diff
@@ -50,6 +47,39 @@ You can use :ref:`filesystem-project` for larger datasets.
    If you also have an account on `lsst-dev <https://developer.lsst.io/services/lsst-dev.html>`_,
    you can access your Notebook Aspect home directory via :file:`$HOME/jhome/` from lsst-dev.
 
+Preinstalled subdirectories
+---------------------------
+
+Within the :file:`$HOME` directory a number of directories are currently pre-created for new users.
+
+:file:`~/notebooks`
+
+The :file:`~/notebooks` directory is intended as the top of a subtree within which you can organize your personal notebooks.
+Within this directory, two items are pre-created:
+
+- A clone of the `lsst-sqre/notebook-demo`_ repository is created from Github at :file:`~/notebooks/notebook-demo`, and the ``prod`` branch of this repository is regularly updated from Github in this clone.  See the :doc:`discussion of demo notebooks <../getting-started/notebook-demo>` for more details.
+- You can use the :ref:`~/notebooks/.user_setups <lsst-kernel-user-setups>` file to configure the environment your notebooks run in.
+
+:file:`~/DATA`
+
+The :file:`~/DATA` directory is pre-created, empty, as a place for tutorial notebooks to load input datasets.
+If it is deleted, some notebooks will attempt to re-create it, while others may fail.
+
+:file:`~/WORK`
+
+The :file:`~/WORK` directory is pre-created, empty, as a place for some tutorial notebooks to write outputs (e.g., "rerun" directories).
+If it is deleted, these notebooks may fail.
+
+:file:`~/dask`
+
+The :file:`~/dask` directory is pre-created and holds an automatically updated `Dask <https://dask.org/>`_ configuration file, :file:`~/dask/dask_worker.yml`.
+This file is recreated on each login, populated with a template that builds dask nodes of the appropriate size and with the appropriate disk mounts.
```

Comment:
```suggestion
This file is recreated on each login and includes a template that builds dask nodes of the appropriate size and with the appropriate disk mounts.
```

(the "automatically updated" phrase is redundant with the second sentence; I like the latter since its more useful)

Reply:
The language was taken verbatim from @athornton's https://lsstc.slack.com/archives/C8EEUGDSA/p1568310501208700 .  I'm not sure if he really meant it _is/contains_ a template or it is created _from/using_ a template; I assumed the latter.  So let's see what he says.

Reply:
It is created from a template; it _is_ a file that defines the default dask worker, as being the same machine image you're currently running, with the same disk mounts and the same CPU/RAM sizing.  You're free to change it yourself, but if it breaks you get to keep both pieces.